ptd_client_server/server.template.conf

```diff
@@ -28,10 +19,6 @@ outDir: D:\ptd-logs\
 # A path to PTDaemon executable binary.
 ptd: D:\PTD\ptd-windows-x86.exe
 
-# A path to a logfile that PTDaemon produces (`-l` option).
-# Note that in the current implementation this file is considered temporary
-# and may be overwritten.
-logFile: logs_ptdeamon.txt
```

Comment:
this is to accommodate new file needed for Uncertainty check?

Reply:
Thank you @dmiskovic-NV for the review. 
This is not related to the new check. This logfile entry is not used even by the current `server.py` script and it is reported as a warning.